appeals/web/src/server/lib/display-page-formatter.js

```diff
@@ -0,0 +1,116 @@
+import logger from '#lib/logger.js';
+import { buildHtmSpan } from '#lib/nunjucks-template-builders/tag-builders.js';
+export const formatDocumentActionLink = (
+	/** @type {any} */ appealId,
+	/** @type {any} */ listOfDocuments
+) => {
+	// If list of documents is in the correct format
+	if (
+		listOfDocuments != null &&
+		typeof listOfDocuments === 'object' &&
+		'documents' in listOfDocuments
+	) {
+		if (Array.isArray(listOfDocuments.documents) && listOfDocuments.documents.length > 0) {
+			return `/appeals-service/appeal-details/${listOfDocuments.documents[0].caseId}/documents/${listOfDocuments.documents[0].folderId}/upload/${listOfDocuments.documents[0].id}`;
```

Comment:
This would be more readable as:

```
const doc = listOfDocuments.documents[0];
return `/appeals-service/appeal-details/${doc.caseId}/documents/${doc.folderId}/upload/${doc.id}`;
```